torch/quantization/fx/quantization_patterns.py

```diff
@@ -327,8 +327,10 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                     scale_node, zero_point_node = create_qparam_nodes(quantizer, self.conv_node.name, scale, zero_point)
                     qconv_args = (conv_input, packed_weight, scale_node, zero_point_node)
                     kwargs = load_arg(quantized=False)(self.conv_node.kwargs)
-                    return quantizer.quantized_graph.create_node(
+                    op = quantizer.quantized_graph.create_node(
                         'call_function', qconv_op, qconv_args, kwargs)
+                    quantizer.node_name_to_scope[op.name] = quantizer.node_name_to_scope[self.conv_node.name]
```

Comment:
can you add a comment here?

Reply:
Actually can you add a TODO here:

"TODO: may need to change the key to Node regenerate the map in each transformationo since we might not be able to rely on the name"